frontend/schemaCMS/src/routes/project/createDataSource/createDataSource.component.js

```diff
@@ -26,6 +27,11 @@ export class CreateDataSource extends PureComponent {
     createDataSource: PropTypes.func.isRequired,
   };
 
+  validateFile = values =>
+    cond([[equals('file'), always(!values.fileName)], [equals('googleSheet'), always(!values.googleSheet)]])(
```

Comment:
we have constants for 'file' and 'googleSheet', please use them

Reply:
fixed